AutoHotkey/Numpad Hotkeys.ahk

```diff
@@ -419,7 +427,7 @@ return
 ::00,26::^{z}
 */
 
-; /* and This is Ctrl letters with the same codes as above
+/* and This is Ctrl letters with the same codes as above

```

Comment:
disabled `ctrl+alpha` set bc it interfered with typing numbers. I may change them to a different layout.